projects/1.18/assets/croptopia-additions/croptopia_additions/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "item.croptopia_additions.rice_flour": "粘米粉",
+  "item.croptopia_additions.curry_powder": "咖喱粉",
+  "item.croptopia_additions.pad_thai": "泰式炒河粉",
+  "item.croptopia_additions.cherry_soda": "樱桃苏打",
+  "item.croptopia_additions.cotton_candy": "棉花糖",
+  "item.croptopia_additions.cola": "可乐",
+  "advancements.glizzy_gobbler.title": "Glizzy Gobbler",
```

```suggestion
  "advancements.glizzy_gobbler.title": "香肠狂热",
```
这个词组稍稍有点荤段子的意思在
不过单纯叫“狼吞虎咽香肠热狗的人”也说得过去